pkg/rest/apisurface.go

```diff
@@ -174,7 +175,47 @@ func (s *APISurface) LastOperationHandler(w http.ResponseWriter, r *http.Request
 		return
 	}
 
-	// TODO
+	request, err := unpackLastOperationRequest(r)
+	if err != nil {
+		// TODO: This should return a 400 in this case as it is either
```

Comment:
Seen these error codes in the OSB spec, maybe we should just double check we are sending the correct status codes in the response, first glance I don't think we are, but just left a TODO for later. :) Same for the one below.

Reply:
Ah yes, that was basically a pseudo-code stopgap, not intended to represent anything correct.